lib/bank_account/checking_account.rb

```diff
@@ -0,0 +1,36 @@
+module BankAccount 
+  class CheckingAccount < Account
+    attr_accessor :checks
+
+    def withdraw(amount)
+    amount += 1
+    if @balance - amount <= 0
+      puts "You cannot withdraw THAT much money."
+    else
+      @balance -= amount
+    end
+    return @balance
+    end
+
+    def withdraw_using_check(amount)
+      @checks += 1
```

Comment:
This is a lot of hard-code values, so this would be a great place to use constants